test/functional/inspec_exec_test.rb

```diff
@@ -395,6 +396,10 @@
       tags_assert = {"password"=>nil, "password-updated"=>nil}
       override['tags'].must_equal tags_assert
       child_profile['parent_profile'].must_equal 'wrapper-override'
+
+      # check for original code on child profile
+      assert = "control 'pro1-con2' do\n  impact 0.9\n  title 'Profile 1 - Control 2'\n  desc 'Profile 1 - Control 2 description'\n  tag 'password'\n  describe file('/etc/passwdddddddddd') do\n    it { should exist }\n  end\nend\n"
```

Comment:
Nooooooo 80 columns!!!...but this is fine...

Reply:
😢